build.sh

```diff
@@ -13,6 +13,13 @@ pushd simulator
 cargo build --release --no-default-features
 popd
 
+# Ensure that examples files are up to date
+./generate_drawing_examples.sh
+git diff --quiet doc/ || (
+    echo "doc/ folder is not up to date"
+    echo "Try running ./generate_drawing_examples.sh"
```

Comment:
IMO this would also be a good place to mention that you should run the montage script if any image has changed.

Reply:
Good idea. I added another line.